<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LowerThirdSwitcher</class>
 <widget class="QDockWidget" name="LowerThirdSwitcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lower Third Switcher</string>
  </property>
  <widget class="QWidget" name="mainLayout">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <property name="horizontalSpacing">
     <number>-1</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>-1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="font">
         <font>
          <italic>false</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Play Next Lower Third</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../images/green-icon.png</normaloff>../images/green-icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>10</height>
         </size>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="countdownTypeTabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="settings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>-1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="mainTextSourceDropdownLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Main Text Source</string>
            </property>
            <property name="margin">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QComboBox" name="secondaryTextSourceDropdownList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="secondaryTextSourceDropdownLabel">
            <property name="text">
             <string>Secondary Text Source</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="groupDropdownLabel">
            <property name="text">
             <string>Lower Third Folder</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLabel" name="displayTimeLabel">
            <property name="text">
             <string>Display Time (milliseconds)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="groupSourceDropdownList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="mainTextSourceDropdownList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QSpinBox" name="displayTimeSpinBox">
            <property name="maximum">
             <number>9999999</number>
            </property>
            <property name="value">
             <number>3500</number>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="sceneDropdownList"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="sceneDropdownLabel">
            <property name="text">
             <string>Scene</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="data">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>-1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Secondary Text</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="mainTextLineEdit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>-1</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="addNewItemButton">
              <property name="text">
               <string>Add new item</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="activeItemButton">
              <property name="text">
               <string>Active</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../images/red-icon.png</normaloff>../images/red-icon.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>10</width>
                <height>10</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteItemButton">
              <property name="text">
               <string>Delete item</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="secondaryTextLineEdit">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QListWidget" name="itemsListWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::IgnoreAction</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>8</width>
              <height>8</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Main Text</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
